src/components/Nav/index.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Drawer, AppBar, Box, Toolbar, IconButton, Typography, Container, Avatar, Tooltip, Button } from '@mui/material';
+import TextField from '@mui/material/TextField';
+import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
+import logoLightMode from '../../assets/images/fit_trax_app_logo_lightmode.png';
+import { avatarStyle, drawerStyle, iconButtonStyle, navStyle, logoStyle, boxStyle, buttonStyle, drawerPaperStyle } from './styles';
+
+const Nav = () => {
+  const [openDrawer, setOpenDrawer] = React.useState(false);
+  const [username, setUsername] = React.useState('');
```

Comment:
do these need an initial state thats not an empty string?

Reply:
No they need to be empty for now, until we intergrade with the API and populate the data.